koku/koku/tests_rbac.py

```diff
@@ -458,6 +472,29 @@ def test_get_access_for_user_data_limited(self, mock_get):
             "openshift.cluster": {"read": []},
             "openshift.node": {"read": []},
             "openshift.project": {"read": []},
+            "ibm.account": {"read": []},
+        }
+        self.assertEqual(access, expected)
+        mock_get.assert_called()
+
+    @patch("koku.rbac.requests.get", side_effect=mocked_requests_get_200_no_next_ibm)
+    def test_ninja(self, mock_get):
```

Comment:
Test NINJA!

Reply:
*me trying to pickup a good name for a test*
!()[https://media.giphy.com/media/vVO14Ew5PYPLy/giphy.gif]